Pdf/Test/Unit/Helper/DataTest.php

```diff
@@ -0,0 +1,64 @@
+<?php
+namespace MagentoJapan\Pdf\Test\Unit\Helper;
+
+use Magento\Framework\TestFramework\Unit\Helper\ObjectManager;
+
+/**
+ * Class DataTest
+ * @package MagentoJapan\Pdf\Test\Unit\Helper
+ */
+class DataTest extends \PHPUnit_Framework_TestCase
```

Comment:
Unfortunately PHPUnit version used by Magento has been updated and this would not be compatible anymore with 2.2 and 2.3